catalog/imaging/imaging-transformer-nitf/src/test/java/org/codice/alliance/transformer/nitf/image/ImageInputTransformerTest.java

```diff
@@ -836,4 +846,28 @@ private static void assertDateAttribute(
       assertDateAttribute(reason, attribute, expectedDateTimes);
     }
   }
+
+  public static class NitfValue {
+    private Object value;
+
+    private Object extValue;
+
+    public NitfValue(Object value, Object extValue) {
```

Comment:
:pencil2: are these values nullable? If so, we should annotate them. If not, then we should check that they aren't being supplied null values.

Reply:
Yes, these values can be null. I have updated the constructors to include the nullable annotation.